fireant/queries/finders.py

```diff
@@ -186,13 +190,21 @@ def find_and_replace_reference_dimensions(references, dimensions):
     return reference_copies
 
 
-def find_and_group_references_for_dimensions(references):
+interval_weekdays = {
+    'month': ('week', 4),
+    'quarter': ('week', 4 * 3),
+    'year': ('week', 4 * 13),
```

Comment:
Is this the intended generalisation for leap years you mentioned? I mean to default to 13 * 4 weeks.

Reply:
No, it's not. This is to convert 1 year into 52 weeks. 13 seems arbitrary since 1 month is converted into 4 weeks and there are 12 months, not 13, but generally 52 weeks is accepted as a clean conversion from 1 year, which is 4 * 13. Maybe it's less confusing if I just put 52 there, instead of 4 * 13?

Reply:
The leap year stuff mentioned in the commit message is the removal of the "monkey patch" function in reference_helpers module.